MobileFilter/sections/general_extensions.txt

```diff
@@ -58,6 +58,8 @@ superomsk.ru#$#body { overflow: visible!important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/36188
 trashbox.ru#$#.div_landing {display: block!important;}
 trashbox.ru#$##div_landing_safe.show { display: none!important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/36360
+sutin0831.pixnet.net#$#.pixnet-happix-pilot__sticky-box[data-link*="/redirect/widgets"] { position: absolute!important; left: -3000px!important; }

```

Comment:
As I understand, in case of `.pixnet-happix-pilot__sticky-box` - `{ position: absolute!important; left: -3000px!important; }` was needed due to:
```
.show {
    display: block!important;
}
```
but it seems that `.pixnet-happix-pilot__sticky-box[data-link*="/redirect/widgets"]` has a higher priority and this rule works fine (at least for me):
`sutin0831.pixnet.net##.pixnet-happix-pilot__sticky-box[data-link*="/redirect/widgets"]`

Reply:
Same for me, no need to use css in this case.